pkg/config/config.go

```diff
@@ -33,8 +33,9 @@ type AdditionalDirectory struct {
 
 // Finch represents the configuration file for Finch CLI.
 type Finch struct {
-	CPUs   *int    `yaml:"cpus"`
-	Memory *string `yaml:"memory"`
+	CPUs        *int    `yaml:"cpus"`
+	Memory      *string `yaml:"memory"`
+	CredsHelper *string `yaml:"credsHelper,omitempty"`
```

Comment:
It is sad that the yaml field naming is not consistent now. There is not an official YAML naming pattern though. https://www.w3schools.io/file/yaml-naming-conventions/#:~:text=There%20are%20no%20official%20naming,C%23%20use%20kebab%20style%20naming.

We can either make credHelper consistent to VMType or additional_directories. Should we follow additional_directories and VMType is experiment and temporary?

Reply:
I think for consistency we can do the same as additional_directories since it isn't a required field either.